src/tokenizer.rs

```diff
@@ -45,9 +45,7 @@ impl Tokenizer {
         '.' | ':' => {
           // Baseline condition - only split when we have a following uppercase letter
           // Get next non-space, non-quote character
-          while text_iterator.peek().unwrap_or(&'.').is_whitespace()
-            || text_iterator.peek() == Some(&'\'')
-          {
+          while text_iterator.peek().unwrap_or(&'.').is_whitespace() || text_iterator.peek() == Some(&'\'') {
```

Comment:
This line is quite readable, at 111 chars. Maybe 120 chars is a good max-width?

Reply:
Wow, some quick digging online suggests this is actually a heated topic. 80 is very highly recommended for very disciplined coding. Maybe I should stick to that and forget my love for long lines. Definitely anything over 120 is considered a code smell by a vast majority of people. So I'll shrink to 80 again and see if it looks decent